### ISSUE TITLE
Latest Pysmi 1.5.1 produces MIB compilation error

### ISSUE THREAD

**Issue by umair-zink** [lextudio/pysnmp#133]
### Expected behavior

compiler.addMibCompiler(mibBuilder, sources=['http://mibs.snmplabs.com/asn1/@mib@'])

should work without any errors

### Actual behavior

root@2a66d39a64fa:/usr/app/src# python pysnmp_listen.py
Traceback (most recent call last):
  File "/usr/app/src/pysnmp_listen.py", line 40, in <module>
    compiler.addMibCompiler(mibBuilder, sources=['http://mibs.snmplabs.com/asn1/@mib@'])
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.13/site-packages/pysnmp/smi/compiler.py", line 41, in add_mib_compiler
    raise error.SmiError("MIB compiler not available: %s" % errorMsg)
pysnmp.smi.error.SmiError: MIB compiler not available: cannot import name 'getReadersFromUrls' from 'pysmi.reader.url' (/usr/local/lib/python3.13/site-packages/pysmi/reader/url.py)

### Detailed steps

Not sure if its a bug or not, but I was using the latest pysnmp with pysmi 1.5.0 till last week and they were working fine, but today i built my docker and it pulled up the latest pysmi 1.5.1 and it started giving me the MIB compiler error on the same code which was working last week.

Changing pysmi back to 1.5.0 works fine again.

### Python package information

root@f3bee6b444e3:/# pip list Package            Version ------------------ ----------- boto3              1.35.39 botocore           1.35.39 certifi            2024.8.30 charset-normalizer 3.4.0 idna               3.10 Jinja2             3.1.4 jmespath           1.0.1 MarkupSafe         3.0.1 pip                24.2 ply                3.11 pyasn1             0.6.1 pysmi              1.5.0 pysnmp             7.1.7 python-dateutil    2.9.0.post0 requests           2.32.3 s3transfer         0.10.3 six                1.16.0 urllib3            2.2.3

### Operating system information

Linux Ubuntu

### Python information

3.13.0

### (Optional) Contents of your test script

```python
mibBuilder = builder.MibBuilder()
compiler.addMibCompiler(mibBuilder, sources=['http://mibs.snmplabs.com/asn1/@mib@'])
```


### Relevant log output

```shell
root@2a66d39a64fa:/usr/app/src# python pysnmp_listen.py
Traceback (most recent call last):
  File "/usr/app/src/pysnmp_listen.py", line 40, in <module>
    compiler.addMibCompiler(mibBuilder, sources=['http://mibs.snmplabs.com/asn1/@mib@'])
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.13/site-packages/pysnmp/smi/compiler.py", line 41, in add_mib_compiler
    raise error.SmiError("MIB compiler not available: %s" % errorMsg)
pysnmp.smi.error.SmiError: MIB compiler not available: cannot import name 'getReadersFromUrls' from 'pysmi.reader.url' (/usr/local/lib/python3.13/site-packages/pysmi/reader/url.py)
```

**Comment by lextudio-support** [comment]
New issues are marked as low priority by default. [Becoming our commercial customers](https://www.pysnmp.com/support#commercial-support), and then your reports are handled with higher priority after triage.

**Comment by lextm** [comment]
Should have been fixed in pysmi 1.5.3 release.
